### DISCUSSION TITLE
Not able to see integration tests

### DISCUSSION THREAD

**Discussion by gazalshukla** [cypress-io/cypress#15601] [General]
In my project , i have added 5 tests but I don't see those tests in cypress.

here is the screenshot:
![image](https://user-images.githubusercontent.com/35330885/111923757-b1700b80-8ac6-11eb-8d73-0835fecf9c0b.png)

**Comment by MatiDiez** [comment]
Long story short your test runner is pulling the specs from another folder. This could be either you are running cypress open on the wrong project in the terminal. Or you have another cypress/integration folder. By your screenshot, it looks like you have another cypress folder (below yarn.lock). 

Your test runner is pulling from the default path cypress/integrations/, not from automation-suite/cypress/integration

If you are changing the project structure, check here https://docs.cypress.io/guides/references/configuration#Folders-Files you can configure where your spec files will live
